src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Extensions/NavigationBarSettingsExtensions.cs

```diff
@@ -15,6 +17,19 @@ public static IMvcBuilder AddNavigationBarSettings(this IMvcBuilder builder, ICo
             
             builder.SuppressNavigationSection(NavigationSection.Reservations);
 
+            var featuresConfiguration = configuration
+                .GetSection(ProviderCommitmentsConfigurationKeys.FeaturesConfiguration)
+                .Get<ProviderFeaturesConfiguration>();
+
+            var manageApprenticesV2Toggle =
+                featuresConfiguration.FeatureToggles.SingleOrDefault(x =>
+                    x.Feature == ProviderFeature.ManageApprenticesV2WithoutPrefix);
+
+            if (manageApprenticesV2Toggle == null || !manageApprenticesV2Toggle.IsEnabled)
+            {
+                builder.SuppressNavigationSection(NavigationSection.ManageApprentices);
```

Comment:
Why would you nuke the Manage Apprentices option in the nav bar? Bit extreme? 💣

Reply:
Based the work on this file on the previous toggle that touched it which was reservations. I can take this out if you think it's too much?

Reply:
I'm not clear what the intent is, but removal of the nav bar option can't be it.